src/components/PingChart.vue

```diff
@@ -286,6 +286,7 @@ export default {
 
             .dark &:hover {
                 background: $dark-font-color;
+                color: black;
```

Comment:
Should this perhaps instead be set to `$dark-bg` given we seem to be swapping the colours for text and background on hover.
```suggestion
                color: $dark-bg;
```